dsa_actionkit/aktemplates.py

```diff
@@ -2,12 +2,14 @@
 import os
 import sys
 
-#STATIC_FALLBACK="/static/js/fallback_local" PYTHONPATH=./djanger django-admin runserver --settings=settings
+# STATIC_FALLBACK="/static/js/fallback_local" PYTHONPATH=./djanger django-admin runserver --settings=settings
 
 
 def serve_templates():
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dsa_actionkit.settings")
```

Comment:
is there a reason to nest the settings for the project under the dsa_actionkit directory/app?  That might be best practice, just checking.

Reply:
(maybe this is standard practice in django these days, if so then ignore me!)

Reply:
This is one of the original files from the directory, I just moved it over. Honestly didn't even really look at it haha, I just didn't want to delete anything that we might still need. Updates to this file are just a result of auto formatting that I did